docs/outputs.rst

```diff
@@ -22,207 +21,330 @@ FMRIPrep generates three broad classes of outcomes:
+variance as possible.
+To make this technique more accessible, *fMRIPrep* automaticaly calculates motion parameter
+expansion [Satterthwaite2013]_, providing timeseries corresponding to first *temporal derivatives* of six motion
+parameters, together with their *quadratic terms*, resulting in the total 24 head motion parameters
+(6 standard motion parameters + 6 temporal derivatives of six motion parameters + 12 quadratic terms of 6 motion
+parameters and their 6 temporal derivatives).
+Additionally, *fMRIPrep* returns temporal derivatives and quadratic terms for the ``csf``, ``white_matter``
+and ``global_signal`` to enable applying 36-parameter denoising strategy
+proposed by Satterthwaite et al. [Satterthwaite2013]_.
+
+Derivatives and quadratic terms are stored under column names with suffixes: ``_derivative1`` and powers ``_power2``.
+These were calculated for head motion estimates (``trans_`` and ``rot_``) and compartment signals
+(``white_matter``, ``csf``, and ``global_signal``).
+
+Confounds for outlier detection
+===============================
+
+- ``framewise_displacement`` - is a quantification of the estimated bulk-head motion calculated using
+  formula proposed by [Power2012]_;
+- ``dvars`` - the derivative of RMS variance over voxels (or :abbr:`DVARS (derivative of
+  RMS variance over voxels)`) [Power2012]_;
+- ``std_dvars`` - standardized :abbr:`DVARS (derivative of RMS variance over voxels)`;
+- ``non_steady_state_outlier_XX`` - columns indicate non-steady state volumes with a single
+  ``1`` value and ``0`` elsewhere (*i.e.*, there is one ``non_steady_state_outlier_XX`` column per
+  outlier/volume).
+
+All these confounds can be used to detect potential outlier time points - frames with high motion or spikes.
+Detected outliers can be further removed from time series using methods such as: volume *censoring* - entirely
+discarding problematic time points [Power2012]_, regressing signal from outlier points in denoising procedure,
+or including outlier points in the subsequent first-level analysis when building the design matrix.
+Averaged value of confound (for example, mean ``framewise_displacement``)
+can be added as a regressor in group level analysis [Yan2013]_.
+
+*Spike regressors* for outlier censoring can also be generated from within *fMRIPrep* using
+the command line options ``--fd-spike-threshold`` and ``--dvars-spike-threshold``
+(default: FD > 0.5 mm or DVARS > 1.5). Spike regressors are stored in separate ``motion_outlier_XX``
+columns.
+
+ICA-AROMA confounds
+===================
+
+- ``aroma_motion_XX`` - the motion-related components identified by :abbr:`ICA (independent components analysis)`
+  -:abbr:`AROMA (Automatic Removal Of Motion Artifacts)` (if enabled with a flag ``--use-aroma``) .
+
+.. warning::
+    If you are already using ICA-AROMA cleaned data (``~desc-smoothAROMAnonaggr_bold.nii.gz``),
+    do not include ICA-AROMA confounds during your design specification or denoising procedure.
```

Comment:
Should we note that, because all other confounds were calculated pre-AROMA-denoising, the additional application of these to a AROMA-cleaned dataset is dubious?

Reply:
Yes